docs/frontend/guides/how-we-use-react-query.md

```diff
@@ -54,31 +88,135 @@ have callback functions that are specific to user actions. This allows us to do
 specific Component or Page actions on Success or Error related to the data being
 shown on that Component or Page.
 
+### Writing Mutations
+
+There are two mutation functions, `mutate` or `mutateAsync`. The `mutate` function does not return anything and utilizes React Query's built in error handling. The `mutateAsync` function returns a promise but requires manual error handling. Preference is given to using the `mutate` function because errors are handled.
```

Comment:
**question:** Is that a preference that TanStack recommends or that we are suggesting?

Reply:
These are TanStack's "recommendations" but not following them causes error and inconsistent behavior 🙃

Reply:
Could you mention that in the documentation? I don't want folks to get caught off guard when they don't use the preference method.